### ISSUE TITLE
`cc-input-date`: improve help text & error message

### ISSUE THREAD

**Issue by florian-sanders-cc** [CleverCloud/clever-components#1391]
## Context

The `cc-input-date` component expects a specific format but by default, the help text only says that the value must be a date without specifying the format.

The error message in case the format is invalid says that it should be a date but doesn't provide any example.
